ext/nnstreamer/extra/nnstreamer_python3_helper.cc

```diff
@@ -480,7 +480,8 @@ parseTensorsInfo (PyObject *result, GstTensorsInfo *info)
     Py_SAFEDECREF (shape_dims);
   }
 
-  return 0;
+  /* Validate output tensors info after parsing python object. */
+  return gst_tensors_info_validate (info) ? 0 : -EINVAL;
```

Comment:
An error message explaining this EINVAL would be nice (ml_loge with the contents of info so that developers can debug their pipelines or python code).

Reply:
Updated, please review the changes again.